chartmogul/api/activities_export.py

```diff
@@ -0,0 +1,44 @@
+from marshmallow import Schema, fields, post_load, EXCLUDE
+from ..resource import Resource
+from chartmogul import ArgumentMissingError
+from collections import namedtuple
+
+
+class ActivitiesExport(Resource):
+    """
+    https://dev.chartmogul.com/v1.0/reference#
+    """
+    _path = "/activities_export"
+    _many = namedtuple('ActivitiesExport', ["id", "status", "file_url", "params", "expires_at", "created_at"])
+
+    class _Schema(Schema):
+        # Create
+        start_date = fields.DateTime()
+        end_date = fields.DateTime()
+        type = fields.String()
+
+        # Retrieve
+        id = fields.String()
+        status = fields.String()
+        file_url = fields.String(allow_none=True)
+        params = fields.Dict(allow_none=True)
```

Comment:
Would be nice if we can define Params schema and use it as nested field. For example: https://github.com/chartmogul/chartmogul-python/blob/main/chartmogul/api/customer.py#L67